src/XMLexport.cpp

```diff
@@ -820,8 +820,16 @@ void XMLexport::writeTrigger(TTrigger* pT, pugi::xml_node xmlParent)
             trigger.append_child("mStayOpen").text().set(QString::number(pT->mStayOpen).toUtf8().constData());
             trigger.append_child("mCommand").text().set(pT->mCommand.toUtf8().constData());
             trigger.append_child("packageName").text().set(pT->mPackageName.toUtf8().constData());
-            trigger.append_child("mFgColor").text().set(pT->mFgColor.name().toUtf8().constData());
-            trigger.append_child("mBgColor").text().set(pT->mBgColor.name().toUtf8().constData());
+#if (QT_VERSION >= QT_VERSION_CHECK(5, 14, 0))
```

Comment:
You only need one set of `#if <condition>` ... `#else` ... `#endif` here - the two lines of Qt version specific code can be grouped together!